frontend/src/components/Forms/TeamEventCreditsForm/TeamEventCreditsForm.tsx

```diff
@@ -59,21 +59,18 @@ const TeamEventCreditForm: React.FC = () => {
   };
 
   const submitTeamEventCredit = async () => {
+    const creditsWithHours = teamEvent?.hasHours
+      ? Number(teamEvent?.numCredits || 0) * Number(hours)
+      : Number(teamEvent?.numCredits || 0);
     const getCredits: (attendance: TeamEventAttendance[]) => number = (attendance) => {
       const filteredAttendance = attendance.filter((event) => event.eventUuid === teamEvent?.uuid);
-      const sum = filteredAttendance.reduce(
-        (acc, event) =>
-          acc +
-          (teamEvent?.hasHours
-            ? Number(teamEvent?.numCredits || 0) * Number(hours)
-            : Number(teamEvent?.numCredits || 0)),
-        0
-      );
+      const sum = filteredAttendance.reduce((acc, event) => acc + creditsWithHours, 0);
```

Comment:
Thanks so much on your bug fixes. Though we don't use it much, I noticed something with team events with hourly credits that allows users to submit more than the max or prevent users from submitting. `creditsWithHours` is the total credits for the user's current submission not each event's hours. 

Here is an example of a user submitting more than the max team credits:

https://github.com/user-attachments/assets/5567e4c2-c3e1-4ce9-8dde-9e0c7ac170ff

A similar process can be repeated for a bug preventing users from submitting when below the max threshold.

Reply:
Thanks for catching that! That appears to have been a long standing bug that went unnoticed. I definitely see usecases for this feature this semester so I'm glad we're fixing that now.

Can you double check again that the validation check now handle all the edge-cases you had in mind? I'm not fully sure what

> A similar process can be repeated for a bug preventing users from submitting when below the max threshold.

is referring to. Let me know if I've already addressed it!

Reply:
Ooh, I think I know what you mean. The problem before was that we weren't multiplying the number of hours with the credits correctly. We used the `hours` variable instead of `teamEventAttendance.hoursAttended`. I think that should all be resolved now since we calculate it correctly. Let me know if it's not and I can follow with a PR to fix!